File: FrontEnd/warehouse/src/app/modules/warehouse/fee/models/fee.model.ts

export class FeeModel {
  id: string;
  name: string;
  value: number;
  isEdit: boolean;
  isActive: boolean;
  rowVersion: any;

  constructor(isEdit: boolean) {
    this.id = '';
    this.name = '';
    this.value = 0;
    this.isEdit = isEdit;
    this.isActive = true;
  }
}

File: BackEnd/WareHouse/Services/WareHouse.Service/Models/FeeModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace WareHouse.Service.Models
{
    /// <summary>
    /// Fee model.
    /// </summary>
    public class FeeModel
    {
        public string Id { get; set; }
        
        [Required]
        [MaxLength(200)]
        public string Name { get; set; }

        public decimal Value { get; set; }

        public bool IsActive { get; set; }

        public byte[] RowVersion { get; set; }

        public string CurrentUserId { get; set; }

        public bool IsEdit { get; set; } = false;
    }
}
